languages/wikipron/src/scrape_and_write.py

```diff
@@ -0,0 +1,123 @@
+import wikipron
+from readme_insert import readme_insert
+from datetime import datetime
+from time import sleep
+import requests
+import os
+import json
+
+
+def call_scrape(lang, config, file_extension):
+    retries = 0
+
+    while retries < 10:
+        file = open(f"../tsv/{lang}{file_extension}.tsv", "w")
+        count = 0
+        try:
+            for (word, pron) in wikipron.scrape(config):
+                count += 1
+                print(f"{word}\t{pron}", file=file)
+            file.close()
+            return count
+        except (
+            requests.exceptions.Timeout,
+            requests.exceptions.ConnectionError
+        ) as err:
+            print(
+                "Timeout or connection error detected during scrape.",
+                lang, file_extension,
+                str(datetime.now()),
+                err
+            )
+            # Pause execution for 30 min.
+            sleep(1800)
```

Comment:
30 min seems a bit long. Was there a particular reason for this choice? (I don't know how robust the Wiktionary API endpoints are.) How often did we encounter errors and hit this sleep code?